packages/ui-new/src/composites/wui-tag/styles.ts

```diff
@@ -40,13 +40,13 @@ export default css`
 
   :host([data-size='md']) {
     height: 30px;
-    padding: 0 ${({ spacing }) => spacing[2]};
+    padding: 0 ${({ spacing }) => spacing[2]} !important;
```

Comment:
Why we need to use important? It's a bad parcitce and we should avoid using it if possible